lib/hsa/hsaBeamformHFBOutput.hpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file
+ * @brief Copy HFB Output gpu to host (for N2 concurrent run)
+ *  - hsaBeamformHFBOutput : public hsaCommand
+ */
+
+#ifndef HSA_BEAMFORM_HFB_OUTPUT_DATA_H
+#define HSA_BEAMFORM_HFB_OUTPUT_DATA_H
+
+#include "hsaCommand.hpp"
+
+/**
+ * @class hsaBeamformHFBOutput
+ * @brief hsaCommand for copying HFB output gpu to host.
+ *
+ * This is an hsaCommand that async copy HFB output buffer from GPU
+ * to CPU. It marks the HFB output buffer to be full when done so that
+ * it can be reused.
+ *
+ * @par GPU Memory
+ * @gpu_mem  hfb_output      Output from the FRB pipeline, size 1024x128x1
```

Comment:
Where are the size numbers coming from?  i.e. maybe add something like beams*freq*....   Also shouldn't it be x4 at the end since they are floats?